packages/pluggableWidgets/timeline-web/src/Timeline.editorPreview.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+    GroupByDayOptionsEnum,
+    GroupByKeyEnum,
+    GroupByMonthOptionsEnum,
+    TimelinePreviewProps
+} from "../typings/TimelineProps";
+import TimelineComponent from "./components/TimelineComponent";
+import { createElement } from "react";
+import { BasicItemType, CustomItemType, ItemType } from "./Timeline";
+
+export function preview(props: TimelinePreviewProps) {
+    const structuredEvents = () => {
+        const eventsMap = new Map<string, ItemType[]>();
+        let groupKey = "";
+
+        Array.from({ length: 5 }).forEach(() => {
+            let constructedItem: ItemType;
+            if (!props.customVisualization) {
+                groupKey = getGroupHeaderByType(
+                    props.groupByKey === "day"
+                        ? props.groupByDayOptions
+                        : props.groupByKey === "month"
+                        ? props.groupByMonthOptions
+                        : "year"
+                );
+                constructedItem = {
+                    icon: props.icon,
+                    title: props.title === String.fromCharCode(160) ? "Title" : props.title,
```

Comment:
`props.title === " "` doesnt work?

Reply:
So this made me pull my hairs :) it is non-breaking space, it is not similar to normal ` `

Reply:
😱